util/missingcrypto.txt

```diff
@@ -383,6 +383,8 @@ COMP_expand_block(3)
 COMP_get_name(3)
 COMP_get_type(3)
 COMP_zlib(3)
+_CONF_get_section(3)
+_CONF_get_string(3)
```

Comment:
You could document them internally, in `doc/internal/man3`

Reply:
Is that "could" or "should"?  They are really a low-level internal implementation detail that should not be documented; the real solution is to cleanup and rewrite NCONF in a future release.

Reply:
I think the proper solution is to rewrite and replace NCONF and remove all the lower-level routines.  I will resolve this comment thread and opened #11662.